core/include/log.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2015 Kaspar Schleiser
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @addtogroup  core_util
+ * @{
+ *
+ * @file        log.h
+ * @brief       System logging header
+ *
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ */
+
+#ifndef __LOG_H
+#define __LOG_H
```

Comment:
No l leading underscores in guards anymore.  See https://github.com/RIOT-OS/RIOT/pull/2412#discussion_r24337496